permissions/src/commonMain/kotlin/Permissions.kt

```diff
@@ -0,0 +1,17 @@
+package com.splendo.mpp.permissions
+
+expect class Permissions internal constructor() {
+
+    fun getBluetoothManager(): PermissionManager
+
+    companion object {
+        fun builder(): Builder
+    }
+
+}
+
+open class Builder() {
```

Comment:
Nice.

Reply:
might be even nicer to use the Kotlin builder pattern though

https://medium.com/google-developers/kotlin-ifying-a-builder-pattern-e5540c91bdbe

Reply:
I've made some changes in `builder`, lets discuss this in new PR